IronFrame/Utilities/FirewallManager.cs

```diff
@@ -43,6 +50,87 @@ public void OpenPort(int port, string name)
             firewallPolicy.Rules.Add(firewallRule);
         }
 
+        internal static string GetFormattedLocalUserSid(string windowsUsername)
```

Comment:
This is internal because it's being used in the test `FirewallManagerTests.cs:26`. I'm usually against exposing functionality to aid tests, but in this case it's internal which makes me feel better. What do you think ?